training/coqui_stt_training/util/config.py

```diff
@@ -546,9 +546,10 @@ def check_values(self):
         check_argument("one_shot_infer", c, is_path=True)
 
 
-def initialize_globals():
-    c = _SttConfig()
-    c.parse_args(arg_prefix="")
+def initialize_config_globals(c):
```

Comment:
It looks like you can implement the initialize_config_globals in the def _post_init_() function of _SttConfig if you want to make things even shorter.